src/plugins/vis_type_timelion/public/components/series/area.tsx

```diff
@@ -19,6 +21,16 @@ interface AreaSeriesComponentProps {
 const isShowLines = (lines: VisSeries['lines'], points: VisSeries['points']) =>
   lines?.show ? true : points?.show ? false : true;
 
+const getPointFillColor = (points: VisSeries['points'], color: string | undefined) => {
+  const pointFillColor = points?.fillColor || points?.fill === undefined ? 'white' : color;
```

Comment:
nit: `points?.fill === undefined` not clear condition. What exactly you want to check for `undefined`: `points`, `points.fill `or both of them

Reply:
I want to check points.fill, but points object can be undefined because of this I should use this construction